test/main.cpp

```diff
@@ -0,0 +1,140 @@
+#include "University.hpp"
+#include "../argparse/argparse.h"
+
+using namespace argparse;
+
+std::vector<std::string> splitStringBySpace(const std::string& str)
+{
+    std::vector<std::string> result;
+    std::istringstream iss(str);
+    std::string word;
+    while (iss >> word) {
+        result.push_back(word);
+    }
+    return result;
+}
+
+int main(int argc, const char **argv)
+{
+    ArgumentParser parser("UniversityTimetable", "University Timetable System");
+
+    parser.add_argument("--addInstructor", "addInstructor")
+        .description("Adds a new instructor by their name, preferred course and timeslot")
+        .required(false);
+
+    parser.add_argument("--addCourse", "addCourse")
+        .description("Adds a new course by its name and preferred time")
+        .required(false);
+
+    parser.add_argument("--addTimeslot", "addTimeslot")
+        .count(3)
+        .description("Adds a new timeslot by its day, start time and end time")
+        .required(false);
+
+    parser.add_argument()
+        .name("--schedule")
+        .description("Generate schedule")
+        .required(false);
+
+    parser.enable_help();
+    auto err = parser.parse(argc, argv);
+    if (err)
+    {
+        std::cout << err << std::endl;
+        return (-1);
```

Comment:
1. Your program will always return 0 
2. Remove brackets

Reply:
1.you mean it will never get into that if?
2.but it contains 2 lines

Reply:
minus was accidentally written

Reply:
can I just return 1 or I need to return different type of errors